requirements.txt

```diff
@@ -1,2 +1,3 @@
 flask
-peewee
+a hunter api key
+that's it
```

Comment:
`requirements.txt` is the file used by `pip` to download dependencies!

Reply:
Fuck lmao i didn't know that